services/management/ia/alerts_service.go

```diff
@@ -116,12 +116,19 @@ func (s *AlertsService) ListAlerts(ctx context.Context, req *iav1beta1.ListAlert
 				return err
 			})
 			if e != nil {
+				// Only FindRuleByID from transaction above can return codes.NotFound
+				if st, ok := status.FromError(e); ok && st.Code() == codes.NotFound {
```

Comment:
Maybe it's time to define ErrNotFound in the models pkg and start using it instead of grpc status error?

Reply:
Currently, we have a custom string for each NotFound error. If we will define package-level error, then its message always will be the same.

Reply:
For such cases you can use `errors.Wrap()` e.g. `errors.Wrap(ErrNotFound, "item not found")`. As a result you will get `item not found: not found`.

Reply:
In that case, the returned error will be not a gRPC error. Also, the message is "inside-out".

Reply:
And then you handle model error and convert it to appropriate gRPC error. 
My main point here is it's a bit weird to handle gRPC error that returned from the storage layer.

Reply:
Well, maybe you right, but then we should refactor all other similar places. Let's leave it for the next time, such refactoring should be done in separate PR.

Reply:
resolving thread for now